lib/ansible/module_utils/facts/packages.py

```diff
@@ -79,5 +79,8 @@ def __init__(self):
         super(CLIMgr, self).__init__()
 
     def is_available(self):
-        self._cli = get_bin_path(self.CLI, False)
+        try:
+            self._cli = get_bin_path(self.CLI)
+        except ValueError:
+            self._cli = None
         return bool(self._cli)
```

Comment:
This is fine for porting but for greenfield code or when someone next cleans this code up, it would be better to write it more like this:

``` python
except ValueError:
    return False
return True
```

Another possibility is that this whole function would be deemed unneeded in a refactor.  Now that you've fixed up `get_bin_path()` to be more sane the caller of `is_available()` might just want to catch the ValueError itself.